src/trrntzip.c

```diff
@@ -960,7 +960,7 @@ int main(int argc, char **argv) {
         fprintf(stdout, "-h : show this help\n");
         fprintf(stdout, "-d : strip sub-directories from zips\n");
         fprintf(stdout, "-f : force re-zip\n");
-        fprintf(stdout, "-g : pause when finished\n");
+        fprintf(stdout, "-g : skip interactive prompts\n");
```

Comment:
`-g` sets `qGUILaunch` to `1`.
This is passed to `OpenErrorLog`where it does skip a prompt, but on Windows it also does what was described before your change. Perhaps we need two options here...